app/models/like.rb

```diff
@@ -0,0 +1,9 @@
+class Like < ApplicationRecord
+  after_create :update_likes_counter
+  belongs_to :post
+  belongs_to :author, class_name: 'User'
+
+  def update_likes_counter
+    Post.find(post.id).increment!(:likes_counter)
+  end
```

Comment:
- [ ] As mentioned below let's use the association to increment the likes of the post.